### PR TITLE
Skips flaky test on the remote cluster API

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Skips flaky test on the remote cluster API

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by hanzei** [comment]
Could You please file a jira ticket to ensure we fix this test?

**Review by hanzei — APPROVED** [review]

**Comment by mgdelacroix** [comment]
It should not be necessary, this PR fixes the test: https://github.com/mattermost/mattermost/pull/27548

**Comment by mgdelacroix** [comment]
There is an unrelated test failing in https://github.com/mattermost/mattermost/pull/27548, so let's merge this to fix master while I investigate the new flaky test and fix that branch

File: server/channels/api4/remote_cluster_test.go [hunk 1]
```diff
@@ -14,6 +14,8 @@ import (
 )
 
 func TestGetRemoteClusters(t *testing.T) {
+	t.Skip("Flaky test")
```
Comment (agnivade) [inline server/channels/api4/remote_cluster_test.go:17]: This should have a JIRA ticket number attached to it so that we can look back and fix it later.

**Review by jespino — APPROVED** [review]

**Comment by agnivade** [comment]
> It should not be necessary, this PR fixes the test: https://github.com/mattermost/mattermost/pull/27548

If that PR fixes the problem, then why not merge that directly?

**Comment by mgdelacroix** [comment]
@agnivade an unrelated flaky test popped up on that branch, today it's definitely my lucky day :P

Added the ticket on https://github.com/mattermost/mattermost/pull/27547/commits/e36fa91cbde04cde7f705bebec83d8edb6b7e088

**Pull request merged**